packages/fyllut/src/components/errors/InternalServerErrorPage.tsx

```diff
@@ -0,0 +1,74 @@
+import { BodyShort, Box, Button, Heading, Link, List, Page, VStack } from '@navikt/ds-react';
+import { InnerHtml, useLanguages } from '@navikt/skjemadigitalisering-shared-components';
+import { TEXTS } from '@navikt/skjemadigitalisering-shared-domain';
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { PATHS } from '../../util/paths';
+
+export function InternalServerErrorPage() {
+  const { translate } = useLanguages();
+  const [searchParams, setSearchParams] = useSearchParams();
+  const [correlationId, setCorrelationId] = useState<string | undefined>();
+  const [historyLength, setHistoryLength] = useState<number | null>();
+  const statusCode = '500';
+
+  useEffect(() => {
+    const correlationId = searchParams.get('correlationId');
+    if (correlationId) {
+      setCorrelationId(correlationId);
+      searchParams.delete('correlationId');
+      setSearchParams(searchParams);
+    }
+    if (typeof window !== 'undefined') {
+      setHistoryLength(window.history.length);
+    }
+  }, [searchParams, setSearchParams]);
+
+  return (
+    <Page tabIndex={-1}>
+      <Page.Block width="xl" gutters>
+        <Box paddingBlock="20 8">
+          <VStack gap="16">
+            <VStack gap="12" align="start">
+              <div>
+                <BodyShort size="small" textColor="subtle">
+                  {`${translate(TEXTS.statiske.error.statusCode)} ${statusCode}`}
+                </BodyShort>
+                <Heading size="large" spacing>
+                  {translate(TEXTS.statiske.error.serverErrorTitle)}
+                </Heading>
+                <BodyShort spacing>{translate(TEXTS.statiske.error.serverErrorMessage)}</BodyShort>
+                <BodyShort spacing>{translate(TEXTS.statiske.error.suggestions)}</BodyShort>
+                <List>
+                  <List.Item>
+                    {`${translate(TEXTS.statiske.error.wait)} `}
+                    <Link href="#" onClick={() => location.reload()}>
+                      {translate(TEXTS.statiske.error.reloadPage)}
+                    </Link>
+                  </List.Item>
+                  {historyLength && historyLength > 1 && (
```

Comment:
Kan vi sjekke window.history.length direkte her, istedet for å legge i en state?